jetty-ant/pom.xml

```diff
@@ -10,7 +10,7 @@
   <name>Jetty :: Ant Plugin</name>
  
   <properties>
-    <bundle-symbolic-name>org.eclipse.jetty.ant</bundle-symbolic-name>
+    <bundle-symbolic-name>${project.groupId}.ant</bundle-symbolic-name>
```

Comment:
I think the Ant module is mostly dead, so probably ok to change the osgi bundle name.

Reply:
ok